tests/assembly/cmse.rs

```diff
@@ -10,14 +13,85 @@ pub trait Sized {}
 pub trait Copy {}
 
 // CHECK-LABEL: __acle_se_entry_point
-// CHECK: bxns
+// CHECK: entry_point:
+//
+// Write return argument (two registers since 64bit integer)
+// CHECK: movs r0, #0
+// CHECK: movs r1, #0
+//
+// If we are using hard-float:
+// * Check if the float registers were touched (bit 3 in CONTROL)
+// hard: mrs     r12, control
+// hard: tst.w   r12, #8
+// hard: beq     .LBB0_2
```

Comment:
Question: I forgor if with `FileCheck` you can capture the label, like `[[.BRANCH_TARGET]]` or something, instead of hard-coding basic block label names?

Reply:
Oh that would be more elegant, I'll check!

Reply:
Fixed :+1: